tools/uninstall-atclient.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+sudo rm -f $(cat ../packages/atchops/build/install_manifest.txt)
```

Comment:
Careful about using relative paths in your bash scripts, these will be based on the shell and differ based on pwd. You can use the following to fix that:

```sh
FULL_PATH_TO_SCRIPT="$(realpath "${BASH_SOURCE[0]}")"
SCRIPT_DIRECTORY="$(dirname "$FULL_PATH_TO_SCRIPT")"
```

Then you can orient yourself based on the current script's dir by prefixing your paths with `"$SCRIPT_DIRECTORY"`

Reply:
Thank

Reply:
That script no longer exists